pyfarm/agent/service.py

```diff
@@ -84,18 +87,19 @@ def __init__(self):
         self.services = {}
         self.register_shutdown_events = False
         self.last_free_ram_post = time.time()
-        self.shutting_down = False
-
-        # reannounce_client is set when the agent id is
-        # first set. reannounce_client_instance ensures
-        # that once we start the announcement process we
-        # won't try another until we're finished
-        self.reannounce_client_request = None
-
-        # Setup scheduled tasks
-        self.scheduled_tasks = ScheduledTaskManager()
-        self.scheduled_tasks.register(
-            self.reannounce, config["agent_master_reannounce"])
```

Comment:
It would be replaced with something new that's smaller but does the same thing:

https://github.com/pyfarm/pyfarm-agent/commit/19ccbdd8d380f9339d33dd747e63101c600b37a5

This code here is probably something I'll make into it's own change....